### ISSUE TITLE
Jupyter kernel dies with njobs > 1

### ISSUE THREAD

**Issue by oort77** [tvdboom/ATOM#18]
## Describe the bug

<br>

## To Reproduce

```import pandas as pd
from sklearn.datasets import load_breast_cancer
from atom import ATOMClassifier

X, y = load_breast_cancer(return_X_y=True)
atom = ATOMClassifier(X, y, logger="auto", n_jobs=2, verbose=2)
atom.impute(strat_num="knn", strat_cat="most_frequent", max_nan_rows=0.1)  
atom.encode(strategy="LeaveOneOut", max_onehot=8, frac_to_other=0.05)  
atom.feature_selection(strategy="PCA", n_features=12)

atom.run(
    models=["LR","XGB"],#  "lSVM", "LDA"
    metric="f1",
    n_calls=25,
    n_initial_points=10,
    n_bootstrap=4,
    random_state = 17
)```



<br>

## Expected behavior
<!--
As described above. Works fine with njobs=1.
-->



<br>

## Additional context
<!--
Add any other context or screenshots about the problem here.
-->



<br>

## Version
<!--
Please run the following code snippet and paste the output here:
 
import atom
print(atom.__version__)
-->

**Comment by tvdboom** [comment]
I couldn't reproduce the error on my Windows machine with Python 3.8.3. Can you check a couple of things so that I have  a better understanding of what's happening:
- How much memory does your machine have and how much of that is being used when running the command?
- Does the same error occur when you don't use hyperparameter tuning (`n_calls=0`)?

**Comment by oort77** [comment]
@tvdboom Ubuntu 20.04 remote box, 32GB RAM. Don't know how to check the amount used, but works fine with n_jobs=1.
Setting n_calls=0 doesn't change the behaviour.
`
<< ================== ATOM ================== >>
Algorithm task: binary classification.
Parallel processing with 3 cores.

Dataset stats ==================== >>
Shape: (569, 31)
Scaled: False
Outlier values: 179 (1.3%)
-------------------------------------
Train set size: 456
Test set size: 113
-------------------------------------
|    |     dataset |       train |        test |
| -- | ----------- | ----------- | ----------- |
| 0  |   212 (1.0) |   169 (1.0) |    43 (1.0) |
| 1  |   357 (1.7) |   287 (1.7) |    70 (1.6) |

Fitting Imputer...
Imputing missing values...
Fitting Encoder...
Encoding categorical columns...
Fitting FeatureSelector...
Performing feature selection ...
 --> Applying Principal Component Analysis...
   >>> Scaling features...
   >>> Total explained variance: 0.971

Training ========================= >>
Models: LR, XGB
Metric: f1


Results for Logistic Regression:         
Fit ---------------------------------------------
`

And here it dies.

**Comment by tvdboom** [comment]
I did some googling and it could be related to a conda / MKL problem, see https://github.com/scikit-learn/scikit-learn/issues/5046.

Proposed solutions:

* conda install numpy (I don't think this will solve it but can't hurt to try)
* conda install nomkl
* conda install -c intel mkl
* Try another virtual environment (e.g. venv, virtualenv)

If nothing of this works I'll keep digging, but either way, this is probably not an ATOM issue, but more likely a scikit-learn / conda issue.
